test/test_copyright.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
```

Comment:
Why 2019 for a new file?

Reply:
Hm, good question.  Well, it was copied from an existing file, which is why I left it as 2019.  But I could change that.